src/main/webapp/app/entities/transcript/transcript-update.tsx

```diff
@@ -54,9 +68,8 @@ export const TranscriptUpdate = (props: ITranscriptUpdateProps) => {
       ...transcriptEntity,
       ...values,
       flags: mapIdList(values.flags),
-      ensemblGene: ensemblGenes.find(it => it.id.toString() === values.ensemblGeneId.toString()),
-      // TYPE-ISSUE: Is selectedGeneId a number or a string?
-      gene: genes.find(it => it.id.toString() === selectedGeneId?.toString()),
+      ensemblGene: ensemblGeneValue ? { id: ensemblGeneValue.value } : undefined,
+      gene: geneValue ? { id: geneValue.value } : undefined,
```

Comment:
@bprize15 Addressed your comments. I actually realized that we don't need to pass in the entire gene object eventhough TranscriptDTO is asking for a `Gene` type. You can just pass in an object with id set to the gene id. 
Actually makes everything simpler.